wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/data/WalletDataSecureStorage.kt

```diff
@@ -178,6 +179,7 @@ class WalletDataSecureStorage private constructor(context: Context) {
 		if (index >= 0) {
 			val item = walletData.removeAt(index) as WalletDataItem.CertificateWalletData
 			val updatedItem = item.copy(pdfData = pdfData)
+			updatedItem.language = language
```

Comment:
This can be moved into the `.copy(pdfData = pdfData, language = language)` call above